go.mod

```diff
@@ -101,13 +101,13 @@ require (
 
 replace (
 	// fixing CVEs
-	github.com/containerd/containerd v1.2.10 => github.com/containerd/containerd v1.4.8
```

Comment:
I wonder why before we were not replacing with `v1.5.10` 🤔  Have you checked if there is any reason?

Reply:
I checked and didn't see any specific reason, seems to have to just missing to do it.